eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+// @ts-check
```

Comment:
Is there any chance the config can be written in TypeScript?

Reply:
It can indeed. Although only through an unstable_ flag in eslint. see [here](https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files)

Pull in the unstable flag (and jiti dev dependency) or leave as js?

Reply:
Hey @kettanaito, Would you be happy to implement in this way?